runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java

```diff
@@ -193,7 +193,12 @@ public void onChatMessage(ChatMessage event)
 
 			if (config.raidsTimer() && message.startsWith(RAID_START_MESSAGE))
 			{
-				timer = new RaidsTimer(spriteManager.getSprite(TAB_QUESTS_BROWN_RAIDING_PARTY, 0), this, Instant.now());
+				timer = new RaidsTimer(null, this, Instant.now());
```

Comment:
This is the only reference to `RaidsTimer` so maybe remove the image from the constructor